File: frontend/src/BuyAndHold/model/RealEstateDTO.ts
import {ExpenseCategoryModel} from "./ExpenseCategoryModel";

export type RealEstateDto = {
    designationOfRealEstate: string,
    roadOfRealEstate: string,
    houseNumberOfRealEstate: string,
    postCodeOfRealEstate: number,
    locationOfRealEstate: string,
    genderOfTenant: string,
    firstNameOfTenant: string,
    lastNameOfTenant: string,
    listOfExpenseCategories: ExpenseCategoryModel[],
    utilityBills: String[]
}

File: backend/src/main/java/de/immoButler/backend/buyAndHold/model/RealEstateDTO.java
package de.immoButler.backend.buyAndHold.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RealEstateDTO {
    private String designationOfRealEstate;
    private String roadOfRealEstate;
    private String houseNumberOfRealEstate;
    private int postCodeOfRealEstate;
    private String locationOfRealEstate;
    private GenderOfTenant genderOfTenant;
    private String firstNameOfTenant;
    private String lastNameOfTenant;
    private List<ExpenseCategory> listOfExpenseCategories;
    private List<String> utilityBills;
}
